gribscan/gribscan.py

```diff
@@ -360,8 +363,7 @@ def write_index(gribfile, idxfile=None, outdir=None, force=False):
             json.dump(record, output_file)
             output_file.write("\n")
 
-    if force or not idxfile.exists():
-        tempfile.rename(idxfile)
+    tempfile.rename(idxfile)
```

Comment:
We should keep this check here and only overwrite if force, as considerable time may have passed since the first check. We should produce a warning, if no rename took place though.